test/e2e/flowvisibility_test.go

```diff
@@ -34,82 +34,76 @@ import (
-    reverseOctetDeltaCountFromDestinationNode: 7083284
-    reverseOctetTotalCountFromSourceNode: 24500996
-    reverseOctetTotalCountFromDestinationNode: 24500996
-    reversePacketDeltaCountFromSourceNode: 136211
-    reversePacketDeltaCountFromDestinationNode: 136211
-    reversePacketTotalCountFromSourceNode: 471111
-    reversePacketTotalCountFromDestinationNode: 471111
-    flowEndSecondsFromSourceNode: 1637706973
-    flowEndSecondsFromDestinationNode: 1637706973
-    throughput: 15902813472
-    throughputFromSourceNode: 15902813472
-    throughputFromDestinationNode: 15902813472
-    reverseThroughput: 12381344
-    reverseThroughputFromSourceNode: 12381344
-    reverseThroughputFromDestinationNode: 12381344
-    sourcePodLabels: {"antrea-e2e":"perftest-a","app":"perftool"}
-    destinationPodLabels: {"antrea-e2e":"perftest-b","app":"perftool"}
+/* Sample record in ClickHouse table:
+timeInserted: 1637706985
+flowStartSeconds: 1637706961
+flowEndSeconds: 1637706973
+flowEndReason: 3
+sourceTransportPort: 44752
+destinationTransportPort: 5201
+protocolIdentifier: 6
+packetTotalCount: 823188
+octetTotalCount: 30472817041
+packetDeltaCount: 241333
+octetDeltaCount: 8982624938
+sourceIP: 10.10.0.79
+destinationIP: 10.10.0.80
+reversePacketTotalCount: 471111
+reverseOctetTotalCount: 24500996
+reversePacketDeltaCount: 136211
+reverseOctetDeltaCount: 7083284
+sourcePodName: perftest-a
+sourcePodNamespace: antrea-test
+sourceNodeName: k8s-node-control-plane
+destinationPodName: perftest-b
+destinationPodNamespace: antrea-test
+destinationNodeName: k8s-node-control-plane
+destinationServicePort: 0
+destinationServicePortName:
+ingressNetworkPolicyName: test-flow-aggregator-networkpolicy-ingress-allow
+ingressNetworkPolicyNamespace: antrea-test
+ingressNetworkPolicyType: 1
+ingressNetworkPolicyRuleName:
+ingressNetworkPolicyRuleAction: 1
+egressNetworkPolicyName: test-flow-aggregator-networkpolicy-egress-allow
+egressNetworkPolicyNamespace: antrea-test
+egressNetworkPolicyType: 1
+egressNetworkPolicyRuleName:
+egressNetworkPolicyRuleAction: 1
+tcpState: TIME_WAIT
+flowType: 1
+destinationClusterIPv4: 0.0.0.0
+octetDeltaCountFromSourceNode: 8982624938
+octetDeltaCountFromDestinationNode: 8982624938
+octetTotalCountFromSourceNode: 30472817041
+octetTotalCountFromDestinationNode: 30472817041
+packetDeltaCountFromSourceNode: 241333
+packetDeltaCountFromDestinationNode: 241333
+packetTotalCountFromSourceNode: 823188
+packetTotalCountFromDestinationNode: 823188
+reverseOctetDeltaCountFromSourceNode: 7083284
+reverseOctetDeltaCountFromDestinationNode: 7083284
+reverseOctetTotalCountFromSourceNode: 24500996
+reverseOctetTotalCountFromDestinationNode: 24500996
+reversePacketDeltaCountFromSourceNode: 136211
+reversePacketDeltaCountFromDestinationNode: 136211
+reversePacketTotalCountFromSourceNode: 471111
+reversePacketTotalCountFromDestinationNode: 471111
+flowEndSecondsFromSourceNode: 1637706973
+flowEndSecondsFromDestinationNode: 1637706973
```

Comment:
How do we construct this sample? It still has some differences with the current data schema. For example, it seems that we have renamed `destinationClusterIPv4` to `destinationClusterIP`. We also removed all the fields  from `octetDeltaCountFromSourceNode` to `reversePacketTotalCountFromDestinationNode`.

Reply:
Originally it is sample output of IPFIX collector, which I think it's not appropriate to keep it anymore. Given there's no such a convenient way to generate the same output from clickhouse DB, do you think we'd better remove it?

Reply:
Not sure whether keeping it can provide convenience when verifying the records. But after taking a look I think a sample record in ClickHouse can be generated by running a query like
```
SELECT * FROM default.flows LIMIT 1 FORMAT Vertical
```

Reply:
thanks, updated